query/common/src/main/java/org/codice/ddf/admin/common/fields/common/CredentialsField.java

```diff
@@ -35,8 +35,6 @@
 
     public CredentialsField() {
         super(DEFAULT_FIELD_NAME, FIELD_TYPE_NAME, DESCRIPTION);
-        this.username = new StringField("username");
-        this.password = new StringField("password");
     }
 
     public CredentialsField username(String username) {
```

Comment:
That will probably be the case when we come across another type of credential needed. I think we should leave it as it is now until a change is needed.